util/stmtsummary/evicted.go

```diff
@@ -190,3 +216,221 @@ func (seElement *stmtSummaryByDigestEvictedElement) toEvictedCountDatum() []type
 func (ssMap *stmtSummaryByDigestMap) ToEvictedCountDatum() [][]types.Datum {
 	return ssMap.other.ToEvictedCountDatum()
 }
+
+func (ssbde *stmtSummaryByDigestEvicted) toCurrentDatum() []types.Datum {
+	var seElement *stmtSummaryByDigestEvictedElement
+	ssbde.Lock()
+	defer ssbde.Unlock()
+
+	if ssbde.history.Len() > 0 {
+		seElement = ssbde.history.Back().Value.(*stmtSummaryByDigestEvictedElement)
+	}
+
+	if seElement == nil {
+		return nil
+	}
+	induceSsbd := &stmtSummaryByDigest{
+		schemaName:    "other",
+		digest:        "other",
+		planDigest:    "other",
+		stmtType:      "other",
+		normalizedSQL: "other",
+		tableNames:    "other",
+	}
+	return seElement.toDatum(induceSsbd)
+}
+
+func (ssbde *stmtSummaryByDigestEvicted) toHistoryDatum(historySize int) [][]types.Datum {
+	// Collect all history summaries to an array.
+	ssbde.Lock()
+	defer ssbde.Unlock()
+	seElements := ssbde.collectHistorySummaries(historySize)
+	rows := make([][]types.Datum, 0, len(seElements))
+	induceSsbd := new(stmtSummaryByDigest)
```

Comment:
`induceSsbd` should be initialized as `other_summary`, just like line232 ~ 238.

Add a function for this initialize to avoid duplicate code.

Reply:
changed